assets/js/components/setup/setup-proxy.js

```diff
@@ -75,28 +77,24 @@ function SetupUsingProxy() {
 
 	let title;
 	let description;
-	let startSetupText;
+	const startSetupText = __( 'Sign in with Google', 'google-site-kit' );
```

Comment:
Let's move it to where it's used in the markup.

Reply:
Updated @eugene-manuilov